pkg/security/rules/engine_linux.go

```diff
@@ -31,13 +32,12 @@ func (e *RuleEngine) GetSECLVariables() map[string]*api.SECLVariableState {
 				continue
 			}
 			cgr := ebpfProbe.Resolvers.CGroupResolver
-			containerWorkloads := cgr.GetContainerWorkloads()
-			if containerWorkloads == nil {
-				continue
-			}
 
-			for _, cgce := range containerWorkloads.Values() {
+			cgr.Iterate(func(cgce *cgroupModel.CacheEntry) {
 				cgce.RLock()
+				if cgce.ContainerContext.ContainerID == "" {
+					return
+				}
```

Comment:
Can we move this after the next line to make sure `cgce.RUnlock()` is called?

Reply:
Nice catch, it's fixed :)